.github/workflows/ci.yml

```diff
@@ -220,8 +221,21 @@ jobs:
       - name: Install packages
         run: |
           sudo apt-get update
-          sudo apt-get install build-essential libtool autotools-dev automake pkg-config bsdmainutils curl ca-certificates ccache python3 rsync git procps bison
+          sudo apt-get install build-essential libtool autotools-dev automake \
+               pkg-config bsdmainutils curl ca-certificates ccache rsync git \
+               procps bison python3 python3-pip python3-setuptools python3-wheel
           sudo apt-get install ${{ matrix.packages }}
+          python3 -m pip install setuptools --upgrade
+
+      - name: Install custom lief wheel
+        if: matrix.os == 'ubuntu-18.04'
+        run: |
+          curl -L -O $SDK_URL/$BIONIC_LIEF_WHL
+          python3 -m pip install $BIONIC_LIEF_WHL
```

Comment:
Not that I mind the two lines here, but I think recent versions of `pip` can install from a URL.